pkg/pillar/cmd/downloader/downloader.go

```diff
@@ -460,22 +457,10 @@ func doDownload(ctx *downloaderContext, config types.DownloaderConfig, status *t
 		return
 	}
 
-	// try to reserve storage, must be released on error
-	kb := types.RoundupToKB(config.Size)
-	if ret, errStr := tryReserveSpace(ctx, status, kb); !ret {
-		status.RetryCount++
-		status.HandleDownloadFail(errStr)
-		publishDownloaderStatus(ctx, status)
-		log.Errorf("doDownload(%s): deferred with %s", config.Name, errStr)
-		return
-	}
 	handleSyncOp(ctx, status.Key(), config, status, dst)
 }
 
 func deletefile(dirname string, status *types.DownloaderStatus) {
-	// XXX common routines to determine pathname?
-	dirname = dirname + "/" + status.ImageID.String()
-
 	// XXX delete whole directory?
```

Comment:
Normally this is just a file, right? So the XXX comment might be misleading.

Reply:
Correct. Updating.